gke-topology-scheduler/README.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+This document gives instructions on how to enable topology in GKE clusters on
+A3M & A3U machines clusters.
+
+The general outline for this to be successful is:
+- We add labels for topology to nodes in the cluster with a daemonset
+- We handle pod scheduling with a scheduling daemon
+- Pods with the added scheduleGate are picked up and scheduled
+
+## Prerequisites
+
+For topology awareness to be enabled, a GKE node pool has to be created with
```

Comment:
I don't believe this is a requirement anymore. DWS doesn't use compact placement but still makes this information available. Do we have a check for this in the code still?

Reply:
it might be not a req for GKE >=1.31, let me recheck

Reply:
confirmed, that it's not req for A3U

Reply:
Thanks, it would be strange for anyone to be installing this on a machine type that doesn't have metadata so hopefully the removal of the check causes no issues